README.md

```diff
@@ -185,6 +185,12 @@ See the comments in the source files in `src/3rdparty/md5` for the complete lice
 The fmt implementation in `src/3rdparty/fmt` is licensed under the MIT license.
 See `src/3rdparty/fmt/LICENSE.rst` for the complete license text.
 
+The nlohmann json implementation in `src/3rdparty/nlohmann` is licences under the MIT license.
```

Comment:
```suggestion
The nlohmann json implementation in `src/3rdparty/nlohmann` is licenced under the MIT license.
```

Reply:
Fixed. I copied the spelling from the other lines, so: licen**S**ed